terraform/modules/emr-launcher/s3.tf

```diff
@@ -24,19 +25,70 @@ resource "aws_s3_bucket_object" "instances" {
   tags = merge(var.common_tags, { Name : "${var.name_prefix}-emr-launch-instances" })
 }
 
-resource "aws_s3_bucket_object" "steps" {
+resource "aws_s3_bucket_object" "analytical_env_steps" {
   bucket = var.config_bucket.id
   key    = "emr/aws-analytical-env/steps.yaml"
-  content = templatefile("../../../cluster_config/steps.yaml.tpl", {
+  content = templatefile("../../../analytical_env_cluster_config/steps.yaml.tpl", {
     config_bucket = var.emr_bucket.id
   })
   tags = merge(var.common_tags, { Name : "${var.name_prefix}-emr-launch-steps" })
 }
 
-resource "aws_s3_bucket_object" "configurations" {
+resource "aws_s3_bucket_object" "analytical_env_configurations" {
   bucket = var.config_bucket.id
   key    = "emr/aws-analytical-env/configurations.yaml"
-  content = templatefile("../../../cluster_config/configurations.yaml.tpl", {
+  content = templatefile("../../../analytical_env_cluster_config/configurations.yaml.tpl", {
+    config_bucket                = var.emr_bucket.id
+    log_bucket                   = var.log_bucket
+    proxy_host                   = var.proxy_host
+    full_no_proxy                = var.full_no_proxy
+    hive_metastore_endpoint      = var.hive_metastore_endpoint
+    hive_metastore_database_name = var.hive_metastore_database_name
+    hive_metastore_username      = var.hive_metastore_username
+  })
+  tags = merge(var.common_tags, { Name : "${var.name_prefix}-emr-launch-config" })
+}
+
+# Batch EMR cluster config files
+resource "aws_s3_bucket_object" "batch_cluster" {
+  bucket = var.config_bucket.id
+  key    = "emr/batch-cluster/cluster.yaml"
+  content = templatefile("../../../batch_cluster_config/cluster.yaml.tpl", {
+    log_bucket             = var.log_bucket
+    ami                    = var.ami
+    account                = var.account
+    security_configuration = var.batch_security_configuration
+    costcode               = var.costcode
+    release_version        = var.release_version
+  })
+  tags = merge(var.common_tags, { Name : "${var.name_prefix}-emr-launch-cluster" })
```

Comment:
What is name_prefix here? Should the batch resources have batch-emv-launch-cluster as a name?